unity-renderer/Assets/Scripts/MainScripts/DCL/Analytics/WorldsAnalytics/IWorldsAnalytics.cs

```diff
@@ -0,0 +1,10 @@
+using DCL;
+using static Decentraland.Bff.AboutResponse.Types;
+
+namespace WorldsFeaturesAnalytics
+{
+    public interface IWorldsAnalytics : IService
+    {
+        void OnEnteredRealm(AboutConfiguration current, AboutConfiguration _);
```

Comment:
not sure that we need to expose it

Reply:
Was exposing it for testing, but not needed now anymore :)